opennms-config-tester/src/main/java/org/opennms/netmgt/config/tester/ConfigTester.java

```diff
@@ -60,6 +63,7 @@
 
 	private ApplicationContext m_context;
 	private Map<String, String> m_configs;
+	private final Pattern m_pattern = Pattern.compile("^directory:(.+)$");
```

Comment:
This can be `static` and should be called something more descriptive than `pattern`.

Reply:
done